src/linear_algebra/prod.lean

```diff
@@ -234,6 +237,24 @@ begin
   rw [←prod_map_comap_prod, submodule.prod_bot],
 end
 
+@[simp]
+lemma prod_map_id : linear_map.prod_map (id : M₁ →ₗ[R] M₁) (id : N₁ →ₗ[R] N₁) = id :=
```

Comment:
Do you actually need `M₁`, or can you just use `M` instead to match the rest of the file?

Reply:
I just thought it made more sense to name the first module `M₁` especially for `prod_map_comp`, but if you prefer I can also change it to `M`.

Reply:
Right now it doesn't match `def prod_map (f : M →ₗ[R] M₃) (g : M₂ →ₗ[R] M₄) : (M × M₂) →ₗ[R] (M₃ × M₄)`. It's best to be consistent.